### ISSUE TITLE
Email Notifications

### ISSUE THREAD

**Issue by bilaliz** [dwservice/agent#19]
remote.it just released this feature:

**_Device Offline Notifications - get notified with an email when devices go offline.  You'll even get notified when they are back online_**

I think I mentioned this idea in another thread too. I think it is essential. However, it will be much much better if we can configure when a notification should be triggered. i.e. if the device is offline/online for x number of minutes. where we can change the X to suit our needs.

**Comment by Gauff-Telematics** [comment]
I agree. This would be a very nice and important feature. This should be on the road map.
